samples/CMakeLists.txt

```diff
@@ -129,8 +129,14 @@ set(SAMPLES
     ccompat
     hipComplex
     hipHostMallocSample
+)
+# We skip this test on macOS, because macho does not support EXCLUDE section flags (used by clang-offload-bundler)
```

Comment:
I plan to fix clang-offload-bundler eventually, but I'm new to most of LLVM, mach-o, HIP, OpenCL and chipStar.
So I'll have to understand why this is necessary in the first place / what exactly it does.